llama.cpp

```diff
@@ -3908,7 +3908,7 @@ void llama_sample_tail_free(struct llama_context * ctx, llama_token_data_array *
 
     // Calculate absolute value of second derivatives
     for (size_t i = 0; i < second_derivatives.size(); ++i) {
-        second_derivatives[i] = abs(second_derivatives[i]);
+        second_derivatives[i] = fabsf(second_derivatives[i]);
```

Comment:
And not `std::abs` as the compiler suggested?

Reply:
I used fabsf because that's what we use elsewhere in this repo. grep shows 29 hits for fabsf, and the only instance of std::abs is for an integer in a fprintf call.

Reply:
The `fabsf` occurrences are almost all in C code, where one indeed needs to use `fabsf`. There is a single `fabsf` instance in `llama.cpp`, which I think should be changed to `std::abs` as this is the idiomatic way of obtaining the absolute value of ints of any size and floats/doubles in C++.

But at the end my comment was just a clearly failed attempt to make a joke. I thought it was funny that you had the compiler warning suggesting `std::abs` to then change it to `fabsf` :-)